index.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html lang="en" >
+<head>
+<title>Tic-tac-toe-js</title>
+<!-- Required meta tags -->
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<!-- Bootstrap CSS -->
+<link rel="stylesheet" href="bootstrap/css/bootstrap.min.css" >
+
+<!--font google-->
+<link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+
+<!--main style-->
+<link rel="stylesheet" href="css/style.css">
+
+</head>
```

Comment:
wcięcia w kodzi, jeśli używasz Visual Code Studio to wciskasz kombinację klawiszy lewy alt + lewy shift + f